src/gluonts/itertools.py

```diff
@@ -154,6 +157,41 @@ def __len__(self) -> int:
         return len(self.iterable)
 
 
+@dataclass
+class PickleCached:
+    """A caching wrapper for ``iterable`` using ``pickle`` to store cached
+    values on disk.
+
+    See ``Cached`` for more information.
```

Comment:
I think this gives you the cross reference

```suggestion
    See :class:`Cached` for more information.
```